Docs/introduction/syncing-overview.md

```diff
@@ -0,0 +1,27 @@
+# Syncing Overview
+
+#### Background
+
+Santa can be run and configured without a sync-server. Doing so will enable an admin to configure rules with the `santactl rule` command. Using a sync-server will enable an admin to configures rules and multiple other settings from the sync-server itself. Santa was designed from the start with a sync-server in mind. This allows an admin to easily configure and sync rules across a fleet of macOS systems. This document explains the syncing process.
+
+#### Flow of a full sync
+
+This is a high level overview of the syncing process end to end. For a more a more detailed account of each part, see the respective documents. The santaclt binary can be run in one of two modes, daemon and non-daemon. The non-daemon mode does one full sync and exits. This is the typical way a user will interact with Santa, mainly to force a full sync. The daemon mode is used by santad to schedule full syncs, listen for push notifications and upload events.
```

Comment:
This is a high level overview of the syncing process. For a more a more detailed account of each part, see the respective documentation.